sceptre/cli/helpers.py

```diff
@@ -36,8 +39,9 @@ def decorated(*args, **kwargs):
         try:
             return func(*args, **kwargs)
         except (SceptreException, BotoCoreError, ClientError, Boto3Error,
-                TemplateError) as error:
-            write(error)
+                TemplateError):
+            if "DEBUG_ERRORS" in os.environ:
```

Comment:
This probably should be reflected in the docs if we're going to do this.

Reply:
Also, I think other envs used by sceptre are prefixed by SCEPTRE_ or something like that. You should check into that.